NewForce-Cohort-9/tabloidmvc-stafford-infection-1
# Rsbranch

# Description
I have created the register function for the app so that new users can register their profile. This coincides with the #29 trello ticket https://trello.com/c/dsizqWLn/29-29-register





## Type of change


# Testing Instructions
STEPS TO TEST
1. After making sure you have our repo cloned down into a local repository and have the database added, git fetch --all in the master branch to retrieve the branch named rsbranch that I have pushed up.
2. Git checkout rsbranch to switch to my branch and open up the project (TabloidMVC) in Visual Studio.
3. Then, run the application in a browser window. 
4. You should see the 'Home' page with a navigation bar at the top of the page. If not already logged out, log out (with the link in the navbar).
5. Then, click the register button under the login button.
6. When promted to register a new user, please enter Bill in the first name tab
7. Please enter Clinton in the last name tab
8. Enter BillClinton in the screen name tab, 
9. Enter BillClinton@gmail. com in the email tab. 
10. Once all the values are filled in, click submit.
11. After you submit the values, In visual studio run the database in the SQL Object explorer, you do this by typing "select * from UserProfile"
12.  Check to see if the values you submitted for the register function are recorded in the database.
13. If they are, the register function is working succesfully and you are good to go.
14. If they aren't, give up and cry.



# Checklist:
